DOCUMENTATION.md

```diff
@@ -68,10 +68,10 @@ or use `pm2` by downloading it from npm - `npm i -g pm2` and start the module by
 
 ### Adding a new repo
 - Use "+" at the bottom right corner to add a repo.
-![add-a-repo](https://user-images.githubusercontent.com/65342122/87232632-0eff7480-c3de-11ea-8a9f-f0a6cf9cd6ee.png)<!-- .element style="height:10%; width:10%" -->
```

Comment:
The image is too stretched in and remove the black border callout from the '+' button